packages/frontend/src/views/create-event.vue

```diff
@@ -209,6 +227,12 @@ export default defineComponent({
       selectedTimes.value.splice(index, 1)
     }
 
+    const result = ref('')
```

Comment:
markdown rendered string should just be a computed property that spits out the rendered string, because markdown rendered string doesn't have any other usage other than displaying purpose